src/frontend/src/lib/components/tokens/TokenLogo.svelte

```diff
@@ -25,7 +27,13 @@
 		{color}
 		{ring}
 	/>
-	{#if showNetworkIcon}
+	{#if nonNullish(tokenCount) && tokenCount > 0}
+		<span
+			class="token-count absolute -right-2.5 bottom-0 flex h-6 w-6 items-center justify-center rounded-full border-[0.5px] border-light-grey bg-white text-sm font-semibold"
```

Comment:
`text--[var(--color-secondary)]` instead of `token-count` ?

Reply:
for sure, changed!